pl-l2d44c2/pl-l2d44c2.xml

```diff
@@ -54,21 +54,44 @@
       <div xml:id="pl-l2d44c2">
         <head xml:id="pl-l2d44c2-Hd1e102">Capitulum 2</head>
         <head xml:id="pl-l2d44c2-Hd1e105" type="question-title">An aliquando resistendum sit potestati.</head>
-        <p xml:id="pl-l2d44c2-d1e108"> Hic oritur quaestio non transilienda silentio. Dictum est enim ** supra,-quod potestas
-          peccandi et nocendi non est homini vel dia- bolo niSi a-Deo. Apostolus 3 autem dicit, quod
-          qui potestati re— szstit, Dei ordinationi resistit. Cum ergo diabolo sit potestas ** mali,
-          Dei ordinatione, potestati. eius non esse resistendum videtur. </p>
-        <p xml:id="pl-l2d44c2-d1e111"> Sed sciendum est Apostolum ibi loqui de saeculari pote- state, scdicet rege et principe
-          et huiusmodi; quibus non est resi- stendum in his quae iubet Deus ei ** exhiberi, scilicet
-          in tributis et huiusmodi. Si vero princeps aliquis vel diabolus aliquid e iusse- rit vel
+        
+        <p xml:id="pl-l2d44c2-d1e108"> Hic oritur quaestio non transilienda silentio. Dictum est enim supra, quod potestas
+          peccandi et nocendi non est homini vel dia-bolo nisi a Deo. 
+        <ref>
+          <name>Apostolus<name>
+        <ref>
+          autem dicit, quod
+        <cit>
+          <quote>
+          qui potestati re— szstit, Dei ordinationi resistit.
+          <quote>
+        <cit>
+           Cum ergo diabolo sit potestas mali,
+          Dei ordinatione, potestati eius non esse resistendum videtur. </p>
+          <!--I've gotten this error several times in other files-->
```

Comment:
you're getting errors everywhere because you're not "closing" elements correctly. Once you fix this, most of your errors will resolve.